draft-ietf-tls-tls13.md

```diff
@@ -1370,14 +1372,14 @@ ClientHello (without modification) except:
 - Including a "cookie" extension if one was provided in the
   HelloRetryRequest.
   
-If a server receives a ClientHello at any other time, it MUST send
-a fatal "unexpected_message" alert and close the connection.
+If a server receives a ClientHello at any other time, it MUST terminate
```

Comment:
Can you clarify your question? this is the same requirement as before.

The reason is that TLS 1.3 forbids renegotiation.